src/clinical/service-worker-thread/tasks.ts

```diff
@@ -49,13 +49,13 @@ function extractDataFromDonors(donors: Donor[], schemasWithFields: any) {
   const recordsMap: any = {};
   donors.forEach(d => {
     Object.values(ClinicalEntitySchemaNames).forEach(entity => {
-      let clincialInfoRecords;
+      let clinicalInfoRecords;
       if (entity === ClinicalEntitySchemaNames.REGISTRATION) {
-        clincialInfoRecords = getSampleRegistrationDataFromDonor(d);
+        clinicalInfoRecords = getSampleRegistrationDataFromDonor(d);
       } else {
-        clincialInfoRecords = getClinicalEntitiesFromDonorBySchemaName(d, entity);
+        clinicalInfoRecords = getClinicalEntitiesFromDonorBySchemaName(d, entity);
       }
```

Comment:
thanks for typo fix... would probably write this like:
```
const clinicalInfoRecords = entity === ClinicalEntitySchemaNames.REGISTRATION
    ? getSampleRegistrationDataFromDonor(d)
    : getClinicalEntitiesFromDonorBySchemaName(d, entity);
```

Reply:
Updated